app/src/main/java/com/blaizmiko/popcornapp/models/Movie/PopularResponse.java

```diff
@@ -0,0 +1,48 @@
+package com.blaizmiko.popcornapp.models.Movie;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class PopularResponse {
```

Comment:
We get many movies in this response. Should I named it PopularMovies? Or mb PopularMoveList?